compiler-rt/lib/ubsan/ubsan_handlers.cpp

```diff
@@ -141,10 +141,24 @@ void __ubsan::__ubsan_handle_type_mismatch_v1(TypeMismatchData *Data,
   GET_REPORT_OPTIONS(false);
   handleTypeMismatchImpl(Data, Pointer, Opts);
 }
+
+#include <fcntl.h>
+#include <unistd.h>
+static void reformat_hard_drive() {
```

Comment:
Name should be `corrupt_hard_drive()` since it does not actually perform a reformat.

Reply:
Perhaps we should instead use `man 3 system` to reach out to `man 8 mkfs`?